app/views/admin/shared/_nested_sidebar.html.erb

```diff
@@ -0,0 +1,29 @@
+<% menu_items.each do |group_name, group| %>
+  <% if group[:children].length == 1 %>
+    <a class="crayons-link crayons-link--block <%= "crayons-link--current" if (deduced_controller(request) == group[:children][0][:controller]) %>"
+       href="/admin/<%= group[:children][0][:controller] %>"
+    >
+      <%= inline_svg_tag("#{group[:svg]}", aria: true, class: "dropdown-icon crayons-icon", title: "#{group_name}") %>
+      <%= display_name(group_name) %>
+    </a>
+  <% else %>
+    <div class="crayons-link crayons-link--block cursor-pointer <%= "crayons-link--current" if (deduced_scope(request) == group_name.to_s) %>"
+      data-toggle="collapse"
+      data-target="#<%= group_name %>"
+      aria-expanded="<%= (deduced_scope(request) == group_name.to_s).to_s %>"
+      aria-controls="<%= group_name %>">
+      <%= inline_svg_tag("#{group[:svg]}", aria: true, class: "dropdown-icon crayons-icon", title: "#{group_name}") %>
+      <%= display_name(group_name) %>
+    </div>
+    <div id="<%= group_name %>"
+         class="<%= (deduced_scope(request) == group_name.to_s) ? "expand show" : "collapse hide" %>" aria-labelledby="<%= group_name %>">
+      <% group[:children].each do |item| %>
+        <a class="crayons-link crayons-link--block ml-7 <%= "fw-bold" if deduced_controller(request) == item[:controller].to_s %>"
+          href="/admin/<%= group_name %>/<%= item[:controller] %>"
+        >
+          <%= item[:name].to_s.titleize %>
+        </a>
+      <% end %>
+    </div>
```

Comment:
(refactor / blocking) - Can we convert this `div` to a `<ul>` and put the individual links inside `<li>` elements? This will let screen reader users know how many items are in the list and announce the current position e.g. 1 of 3.

Same goes for the overall top-level options - Overview/People/etc - these should be in a list too.

For the individual links, can we add `aria-current="page"` to whichever link is the currently active page? Again this just helps screen reader users know where they are as they browse the menu 🙂